src/reducers/user.js

```diff
@@ -0,0 +1,34 @@
+import produce from "immer";
+import {
+  LOGIN_PENDING,
+  LOGIN_SUCCESS,
+  LOGIN_FAILURE,
+  LOGOUT
```

Comment:
```suggestion
  LOGOUT,
```
콤마.. ESLint에 규칙 추가된 것 같은데 안 잡네용